### PR TITLE
[SEMIMODULAR] Ghosts and antags can view crew background records

### PR DESCRIPTION
**Pull request by desertmote**
## About The Pull Request

Adding onto exploitables, this allows background records to be read by ghosts or antags, including the TGUI verb found in the OOC tab.

## How This Contributes To The Skyrat Roleplay Experience

Background records are currently unused.

## Changelog

:cl:
add: Ghosts and antags can now view a character's background info
/:cl:

### CODE REVIEW

File: code/datums/datacore.dm [hunk 1]
```diff
@@ -273,6 +273,10 @@ GLOBAL_DATUM_INIT(data_core, /datum/datacore, new)
 			G.fields["past_records"] = C.prefs.read_preference(/datum/preference/text/general)
 		else
 			G.fields["past_records"] = ""
+		if (C && C.prefs && C.prefs.read_preference(/datum/preference/text/background))
```
Comment (Zonespace27) [inline code/datums/datacore.dm]:
```suggestion
		if(C?.prefs?.read_preference(/datum/preference/text/background))
```

File: code/datums/datacore.dm [hunk 2]
```diff
@@ -273,6 +273,10 @@ GLOBAL_DATUM_INIT(data_core, /datum/datacore, new)
...
+			G.fields["background_records"] = C.prefs.read_preference(/datum/preference/text/background)
+		else
+			G.fields["background_records"] = ""
```
Comment (tf-4) [inline code/datums/datacore.dm]:
```suggestion
		G.fields["background_records"] = C?.prefs?.read_preference(/datum/preference/text/background) || ""
```
I get this isn't in keeping with the other bits, but they can be updated too.

**Comment by desertmote** [comment]
What are the thoughts on letting antags see this info too?

**Comment by hugmeorelse** [comment]
> What are the thoughts on letting antags see this info too?

Sure.

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**